irrep/spacegroup.py

```diff
@@ -1148,9 +1183,9 @@ def match_symmetries(self, refUC=None, shiftUC=None, signs=False):
             for i, sym2 in enumerate(self.symmetries_tables):
                 t1 = refUC.dot(sym2.t - t) % 1
                 #t1 = np.dot(sym2.t - t, refUC) % 1
-                t1[1 - t1 < 1e-5] = 0
+                t1[1 - t1 < trans_thresh] = 0
                 if np.allclose(R, sym2.R):
```

Comment:
So, for comparison of rotational parts we use `np.allclose` with default values [`rtol=1e-05, atol=1e-08`](https://numpy.org/doc/stable/reference/generated/numpy.allclose.html), which are quite strict. Can this create a problem in principle? or is it unlikely?

I mean, should we make it another user_defined parameter `rot_thresh`?

Reply:
The difference between the vector for translation and the matrix for rotations is that the matrix elements of the later are integers, because we work with work with a direct basis rather than the cartesian basis. In principle, the difference between the rotational parts in both settings should have integer matrix elements.

Anyway, I will add a comment as a reminder, just in case a user opens an issue with it.